front/components/V2/TabNavigationPhone.tsx

```diff
@@ -0,0 +1,70 @@
+import { View } from 'react-native';
+import { Center, ScrollView } from 'native-base';
+import TabNavigationButton from './TabNavigationButton';
+import { NaviTab } from './TabNavigation';
+
+type TabNavigationPhoneProps = {
+	tabs: NaviTab[];
+	activeTabID: string;
+	setActiveTabID: (id: string) => void;
+	children?: React.ReactNode;
+};
+
+const TabNavigationPhone = (props: TabNavigationPhoneProps) => {
```

Comment:
The name of this component is misleading. It should be either renamed or documented